### DISCUSSION TITLE
Mapping local CA cert to container so it is trusted

### DISCUSSION THREAD

**Discussion by ChrisThePCGeek** [semaphoreui/semaphore#1324] [Q&A]
I have semaphore running in k3s and run a local CA for my homelab environment.  The CA cert needs to be installed to the trusted certs location inside the container so it will trust my gitlab-ce locally hosted as well.  I tried using a configmap and mounting it as a volume to the /etc/ssl/certs folder inside the container as internal-ca.pem and the file is there but the container has not updated it,  I think it needs to run update-ca-certificates but its root only and no root access.  Any help would be appreciated.  I would rather not build my own version of the container to make it easier for upgrades.  Thank you!

**Comment by Niolith** [comment]
A workaround that I did :

`docker exec -it semaphore /bin/sh`
`cat /etc/ssl/certs/ca-certificates.crt`
Exit Container
Create a new file ca-certificates.crt and append your CA certificate at the end
`docker cp /root/ca-certificates.crt semaphore:/etc/ssl/certs/ca-certificates.crt`

Your certificate should be trust
